src/app/components/attestation/attestation-create.jsx

```diff
@@ -135,159 +171,227 @@ function ChplAttestationCreate(props) {
+              </Typography>
+              <Typography gutterBottom>To continue with the electronic signature process, please enter your name and click the &quot;Sign Electronically&quot; button to confirm and submit the Attestations to your ONC-Authorized Certification Body (ONC-ACB) for review.</Typography>
+              <br />
+
+              <div className={classes.userContext}>
+                <Card>
+                  <CardContent>
+                    <div>
+                      <Typography gutterBottom variant="subtitle1">
+                        Name:
+                      </Typography>
+                      <Typography>{user.fullName}</Typography>
+                    </div>
+                  </CardContent>
+                </Card>
+                <Card>
+                  <CardContent>
+                    <div>
+                      <Typography gutterBottom variant="subtitle1">
+                        Title:
+                      </Typography>
+                      <Typography>{user.title}</Typography>
+                    </div>
+                  </CardContent>
+                </Card>
+                <Card>
+                  <CardContent>
+                    <div>
+                      <Typography gutterBottom variant="subtitle2">
+                        Health IT Developer:
+                      </Typography>
+                      <Typography>{developer.name}</Typography>
+                    </div>
+                  </CardContent>
+                </Card>
+              </div>
+              <br />
+              <div className={classes.signDateContainer}>
+                <Card>
+                  <CardContent>
+                    <ChplTextField
+                      id="signature"
+                      name="signature"
+                      label="Electronic Signature"
+                      required
+                      value={signature}
+                      onChange={handleSignature}
+                      helperText="Enter your name"
+                      />
+                  </CardContent>
+                </Card>
+                <Card>
+                  <CardContent>
+                    <Typography gutterBottom variant="subtitle1">
+                      Date:
+                    </Typography>
+                    <Typography variant="body1">
+                    <Moment
+                        date={Date.now()}
+                        format="DD MMM yyyy"
+                      />
+                    </Typography>
+                  </CardContent>
+                </Card>
+              </div>
+              <br />
+              <Button
+                fullWidth
+                id="sign-electronically"
+                variant="contained"
+                color="primary"
+                onClick={handleSubmit}
+                disabled={isSubmitDisabled()}
+              >
+                Sign Electronically
+                <SaveIcon
```

Comment:
Is this the right icon? I was looking for something that suggested a pen, or a signature, or something, but couldn't find something worthwhile. Do you have any better ideas?